airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1237,6 +1237,38 @@ components:
           type: boolean
         securityUpdates:
           type: boolean
+        notifications:
+          type: array
+          items:
+            $ref: "#/components/schemas/Notification"
+    Notification:
+      type: object
+      required:
+        - notificationType
+      properties:
+        # Instead of this type field, we would prefer a json schema "oneOf" but unfortunately,
+        # the jsonschema2pojo does not seem to support it yet: https://github.com/joelittlejohn/jsonschema2pojo/issues/392
+        notificationType:
+          $ref: "#/components/schemas/NotificationType"
+        slackConfiguration:
+          $ref: "#/components/schemas/SlackNotificationConfiguration"
+    SlackNotificationConfiguration:
+      type: object
+      required:
+        - webhook
+      properties:
+        webhook:
+          type: string
+        onFailure:
```

Comment:
what does this field mean? is this a general configuration for a notification or slack specific? i think i'm inclined to say it's a question we don't need to answer now and we can avoid it if we just remove the field entirely.

Reply:
ok, let's remove it for now 

it was coming from https://github.com/airbytehq/airbyte/pull/2703#issuecomment-812302609. 
I am guessing it was to distinguish between success and failures, but since this PR does not handle success notifications yet and it would complicate the UI for the moment, we can figure it out later...